### DISCUSSION TITLE
关于 luci-app-dockerman

### DISCUSSION THREAD

**Discussion by cherinyy** [immortalwrt/immortalwrt#1004] [Q&A]
第一次，我是使用值守式系统升级服务直接向 23.05.0-SNAPSHOT 版本固件添加 luci-i18n-dockerman-zh-cn 以内置 Docker 服务。
然后我发现这会导致设备几乎一分钟重启一次.

于是我回退到使用官方编译的 23.05.0-rc2 - r26681-4fffb9b21a，手动通过 LuCI 界面安装 luci-i18n-dockerman-zh-cn。这次是还没安装完就导致设备重启了，并且不断地重复。

这是有什么冲突了吗？

**Comment by 1715173329** [comment]
你需要问 dockerman
